app/controllers/pizzas_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PizzasController < ApplicationController
+  def new
+    puts params
+    if params[:pizzeria_id]
+      @pizzeria = Pizzeria.find(params[:pizzeria_id])
+      @pizza = Pizza.new
+    else
+      @pizza = Pizza.new
+    end
+  end
+
+  def create
+    @pizza = Pizza.new(pizza_params)
+
+    if @pizza.save
+      redirect_to pizza_path @pizza, notice: 'Pizza was successfully created.'
```

Comment:
If you're not handling the `notice`, don't include it in the redirect. As of now, it does nothing.